doc_src/functional_hazard_analysis_longline/fha_longline.tex

```diff
@@ -355,34 +358,847 @@ \subsection{Phase 1: Approach and Positioning}
+	       	This could put strain on the highline from the falling rope and \textcolor{teal}{rescuer}.
+	       	The problem is that the highline can be subjected to multiple impacts by the falling \textcolor{teal}{rescuer} and rope.
+	       	In addition, the weight of the \textcolor{teal}{rescuer} could pull the rope over the highline and cause damage.}, 
+	     fhaMitigationKeyValue=
+	     	{\tabitem Abrasion-resistant coating of line and backups \\
+	     	\tabitem Line construction with multiple backups made of core-sheath ropes\\
+	     	\tabitem Avoid having \textcolor{teal}{rescuer} and helicopters on different sides of the highline},
+	     fhaCritPostKeyValue=
+	     	{\textcolor{orange}{medium}, as the sheath prevents friction damage to the line. Damage caused by hitting can be mitigated by the backups used.}]
+
+%1.6
+\fhaTable[fhaSystemKeyValue=
+			{Highline},
+		 fhaFunctionKeyValue=
+		 	{positioning},
+	     fhaCritPreKeyValue=
+	     	{\textcolor{red}{high}},
+	     fhaNumberKeyValue=
+	     	{1.6},
+	     fhaCausesKeyValue=
+	     	{Aerodynamic interaction of highline with helicopter downwash (rocking of the line in the wind)}, 
+	     fhaDescriptionKeyValue=
+	     	{Aerodynamic interaction between the helicopter and the highline is possible even before the \textcolor{teal}{rescuer} can reach the \textcolor{purple}{patient}.
+	     	Since long highlines can swing strongly sideways and have little damping in this direction, this can prevent the helicopter from positioning the \textcolor{teal}{rescuer} with sufficient accuracy.
+			Furthermore, the downwash can cause the loops of the highline backup to swing around and get caught in various places on the \textcolor{purple}{patient} or \textcolor{teal}{rescuer}.
+			These could have to be cut during the subsequent rescue process or could lead to unwanted tied-up situations.}, 
+	     fhaMitigationKeyValue=
+	     {\tabitem Termination of the attempt in case of acute rocking of the highline \\
+	     \tabitem Hovering near highline to test aerodynamic interaction (careful approach)\\
+	     \tabitem Sheathing the highline and backups to eliminate loops\\
+	     \tabitem Tighten the highline more when setting up to reduce the amplitude},
+	     fhaCritPostKeyValue=
+	     {\textcolor{orange}{medium}, as the formation of loops is prevented and a previously agreed, slow approach to positioning of the helicopter allows an abort at any time}]
+	     
+%1.7
+\fhaTable[fhaSystemKeyValue=
+			{Highline},
+		 fhaFunctionKeyValue=
+		 	{load transfer},
+	     fhaCritPreKeyValue=
+	     	{\textcolor{red}{high}},
+	     fhaNumberKeyValue=
+	     	{1.7},
+	     fhaCausesKeyValue=
+	     	{Rockfall or other objects blown down by downwash}, 
+	     fhaDescriptionKeyValue=
+	     	{A falling stone, branch, carabiner, etc. is enough to cut the line under tension.
+	      	This causes the \textcolor{purple}{patient} to fall into the backup and hang lower than in the line.
```

Comment:
line-> mainline

Reply:
I chose highline in https://github.com/JBludau/Highline-Rescue-Project/pull/10/commits/3306c8304bc49714804b65a96762ae442ec3d629 ... we did not differentiate the parts of the highline in the fha section of the document ... But it might be a good idea ... 

Would you create an issue to track that?